frontend/packages/employee-frontend/src/assets/i18n/fi.ts

```diff
@@ -288,6 +288,9 @@ export const fi = {
       endTimePlaceholder: '16:00',
       shiftCareLabel: 'Ilta- ja vuorohoito',
       shiftCareValue: 'Tarvitaan ilta- ja vuorohoitoa',
+      shiftCareWithoutAttachments:
+        'Tarvitaan ilta- ja vuorohoitoa (liitteet puuttuu)',
```

Comment:
Since attachments are not in production yet, I don't think using this text is a good idea. You could just use `shiftCareValue` when there are no attachments.